Create vkquake2.json

RavenMacDaddy opened the pull request:
I created this manifest for vkQuake2 - feel free to use it.

Calinou commented:
Could you remove the `yquake2.json` changes from this pull request?

In the future, I recommend using separate branches for each pull request and never using `master` as a source for PRs :slightly_smiling_face:

RavenMacDaddy commented:
I see, thank you for notifying me.

Right as I removed it it failed on appveyor, is there something I need to fix or is this erroneously simply because I removed the file from the PR?

Also, when you say use separate branches for each pull request, does that mean temp1 for vkquake2 and temp2 for yquake2 as an example or simply something other than master?

Calinou commented:
Superseded by https://github.com/Calinou/scoop-games/pull/233.
